CHANGELOG.md

```diff
@@ -1,21 +1,64 @@
+## v1.2.0 (04/02/2020)
+
+**Common**
+
+* Added a protected read-only cache. For details, see `CacheSettings`.
+* Added a protected read-only cache [usage example](https://github.com/heremaps/here-olp-sdk-cpp/blob/master/docs/dataservice-cache-example.md).
```

Comment:
This will not work forever. Probably locking to a specific commit is better, e.g. https://github.com/heremaps/here-olp-sdk-cpp/blob/c30c4051ec012eaca09b7aa7ba4c7e8f6cd7a576/docs/dataservice-cache-example.md
Last commit before this change.